apis/nucleus/src/viz.js

```diff
@@ -253,25 +278,7 @@ export default function viz({
       await rendered;
       return cellRef.current.takeSnapshot();
     },
-    // ===== undocumented experimental API - use at own risk ======
-    /**
-     *  valid types: viewData, cssScaling, snapshot, exportData, exploration
-     *  questionable types: supportRefresh, quickMobile, fullscreen
-     *  deprecated?: sharing
-     *
-     */
-    support(type) {
-      if (mountedReference && successfulRender) {
-        return cellRef.current.support(type, originalExtensionDef?.support, originalLayout);
-      }
-      return false;
-    },
-    toggleFocus(focus) {
-      cellRef.current.toggleFocus(focus);
-    },
-    setOnBlurHandler(cb) {
-      cellRef.current.setOnBlurHandler(cb);
-    },
+    ...newExperimental,
```

Comment:
I did this as well to guard against unnessecary usage before we are ready to set it as a public API.
@linhnguyen-qlik @T-Wizard

Reply:
And all the function inside newExperimental which you spread here will get the warning? Great I think.
*  valid types: viewData, cssScaling, snapshot, exportData, exploration
       *  questionable types: supportRefresh, quickMobile, fullscreen
       *  deprecated?: sharing

Reply:
There is no warning, they are basically not present if you don't pass in the context option. Mainly want to do this to stop people from seeing the functions when debugging and then using them before we make them official.